SECURITY.md

```diff
@@ -11,9 +11,13 @@ __How?__ Inside affected repository --> security tab
 for contributor:
 --> Report a vulnerability
 
+Open a draft security advisory : https://github.com/eclipse-tractusx/sig-release/security/advisories/new
+
 for committer:
 --> advisories --> New draft security advisory
 
+Open a draft security advisory : https://github.com/eclipse-tractusx/sig-release/security/advisories/new
+
```

Comment:
Why the change? TRG 7.01 shows how the content of the SECURITY.md file should look like and the current state is correct:
https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01#security-file

Reply:
This is the way how CVS are reported in Tractus-X. So this link is important to be added to enable external contributors to reports CVS in the correct way.

Reply:
that's right, but the link points to the sig-release project, which is not correct and recommends to open a advisory even for contributors which is not possible. 
The recommended security file already contains all necessary information, so IMO we should not change this